html/captive-portal/lib/captiveportal/PacketFence/Controller/Node/Manager.pm

```diff
@@ -31,7 +31,7 @@ sub unreg :Local :Args(1) {
     my $node = node_view($mac);
     if ($username && $node) {
         $c->log->info("$username attempting to unregister $mac");
-        if ($username ne $default_pid && $username eq $node->{pid}) {
+        if (($username ne $default_pid || $username ne $admin_pid ) && $username eq $node->{pid}) {
```

Comment:
I am actually testing the pull-request review! You pass the test ! ;)
Let me fix that for you!